contracts/BC_fusion/StakeHub.sol

```diff
@@ -680,6 +684,10 @@ contract StakeHub is System, Initializable {
         uint256 jailUntil = block.timestamp + downtimeJailTime;
         _jailValidator(valInfo, jailUntil);
 
+        if (IStakeCredit(valInfo.creditContract).getPooledBNB(operatorAddress) < minSelfDelegationBNB) {
```

Comment:
Do we need this condition? I think even it get enough delegation, we should remove it.

Reply:
Gracias